Riot/Assets/en.lproj/Vector.strings

```diff
@@ -205,7 +205,7 @@
 "room_participants_unknown" = "Unknown";
 "room_participants_idle" = "Idle";
 "room_participants_now" = "now";
-"room_participants_ago" = "ago";
+"room_participants_since" = "since";
```

Comment:
Could we allow even more freedom by adding formatting within the translation so that languages are free to apply their right wording order?

Like:
```suggestion
"room_participants_ago_format" = "%@ ago";
```

(Please keep `ago`)